pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -235,6 +272,8 @@ func (rq *ResourceQuotaController) Run(workers int, stopCh <-chan struct{}) {
 	glog.Infof("Starting resource quota controller")
 	defer glog.Infof("Shutting down resource quota controller")
 
+	go rq.quotaMonitor.Run(stopCh)
```

Comment:
Move it after `WaitForCacheSync()`

Reply:
I think the order is ok since it waits for informers to start.  @derekwaynecarr that's your thinking too?

Reply:
yes